manifests/pkcs11_eventmgr.pp

```diff
@@ -32,46 +32,63 @@
 #
 #   Default: {}
 #
+# @param event_opts_base
+#   Hash
+#
+#   Default: $pam_pkcs11::params::pkcs11_event_opts
+#
 # @param autostart_method
 #   String
 #
 #   Default: $pam_pkcs11::params::pkcs11_eventmgr_autostart_method,
 #
 class pam_pkcs11::pkcs11_eventmgr (
-  Boolean    $debug            = false,
-  Boolean    $daemonize        = true,
-  Integer[0] $polling_time     = 1,
-  Integer[0] $expire_time      = 0,
-  String     $pkcs11_module    = 'default',
-  Hash       $event_opts       = {},
-  String     $autostart_method = $pam_pkcs11::params::pkcs11_eventmgr_autostart_method,
+  Boolean                 $debug            = false,
+  Boolean                 $daemonize        = true,
+  Integer[0]              $polling_time     = 1,
+  Integer[0]              $expire_time      = 0,
+  Variant[Enum['default'],
+  Stdlib::AbsolutePath] $pkcs11_module      = 'default',
```

Comment:
This linebreak without additional indentation looks a bit weird to me. Is this autoformatted? If not, I'd keep this on one line or add indentation on the second one.

Reply:
Yes. puppet-lint is strange with these things, I noticed as well. Not sure how to change these things in puppet-lint... These are the default settings of puppet-lint from the PDK.